projects/plugins/crm/includes/ZeroBSCRM.Core.php

```diff
@@ -1819,10 +1819,18 @@ public function init() {
 
 			// } Catch front end loads :)
 			if ( $this->settings->get( 'killfrontend' ) == 1 ) {
+				global $pagenow;
 
 				// http://wordpress.stackexchange.com/questions/12863/check-if-were-on-the-wp-login-page
 				// } 2.0.2 also allow /wild stuff (e.g. welcome wiz)
```

Comment:
Should we update this comment? It seems like we're not only checking for the login page anymore 😄

Reply:
Do you mean the line `1824` or both comments, `1824` and 1825`?

Reply:
Removed both! (clean code)